azure-pipelines.yml

```diff
@@ -280,10 +286,11 @@ stages:
         env:
           ARTIFACTORY_DEPLOY_USERNAME: $(ARTIFACTORY_DEPLOY_USERNAME)
           ARTIFACTORY_DEPLOY_PASSWORD: $(ARTIFACTORY_DEPLOY_PASSWORD)
+          PGP_PASSPHRASE: $(PGP_PASSPHRASE)
```

Comment:
This is not propagated by default. We have to copy it by hand (similar with `ARTIFACTORY_DEPLOY_USERNAME` and `ARTIFACTORY_DEPLOY_PASSWORD`).